hyperspy/_components/heaviside.py

```diff
@@ -29,10 +29,12 @@ class HeavisideStep(Expression):
     .. math::
 
         f(x) =
-        \\begin{cases}
-        0 & x<n\\\\
-        A & x>=n
-        \\end{cases}
+        \begin{cases}
```

Comment:
Is it worth documenting that this is using the `half-maximum convention` as defined by numpy?

Reply:
Good point. I also added the table of variables we have for other components.